tools/pipelines/templates/include-set-package-version.yml

```diff
@@ -144,3 +153,21 @@ steps:
         echo "##vso[task.logissue type=error]Fluid internal dev versions shouldn't use caret dependencies"
         exit -1;
       fi
+
+- ${{ if ne(parameters.packageTypesOverride, 'none') }}:
+  - template: include-install-pnpm.yml
+    parameters:
+      buildDirectory: ${{ parameters.buildDirectory }}
+  - task: Npm@1
+    displayName: npm run build:docs
+    inputs:
+      command: 'custom'
+      workingDirectory: ${{ parameters.buildDirectory }}
+      customCommand: 'run build:docs'
```

Comment:
Why are these two steps needed? pnpm and flub are already installed earlier, and build:docs is done during the build. If you run it here, then it will end up running `tsc` as well, because that's required to build the docs. I think it's better to move the setting of the package types to immediately after setting the version, and don't run build:docs manually. That way the regular build will have the updated version and types.

Reply:
I have moved `Set Package Types` after Build stage. The reason being the flub command checks if the path exists: https://github.com/microsoft/FluidFramework/blob/main/build-tools/packages/build-cli/src/commands/release/setPackageTypesField.ts#L156. Before Build stage, rollups are not generated. So there are two ways:
1. Run build:docs followed by Set Package Types and then Build stage.
2. Run Build stage, Set Package Types and build again with the updated types.